capstone/src/main/java/com/google/univiz/CarnegieSizeDegree.java

```diff
@@ -27,8 +27,13 @@
   SIZE18("Exclusively graduate and professial institution");
 
   private final String degree;
+  private static CarnegieSizeDegree[] degreesList = CarnegieSizeDegree.values();
 
   CarnegieSizeDegree(String degree) {
     this.degree = degree;
   }
+
+  public static CarnegieSizeDegree getDegree(int index) {
+    return degreesList[index];
```

Comment:
note that I don't think `values()` ordering is guaranteed to be deterministic. If this works after you've added some tests, let's go with it. If not we can add an int into the enum to make the ordering explicit.

Reply:
I've added a [note to your enum PR](https://github.com/googleinterns/univiz/pull/66/files#r468147441) which is the implementation for Chris's suggestion here.